app/assets/javascripts/discourse/app/models/post.js

```diff
@@ -352,6 +352,32 @@ const Post = RestModel.extend({
     }
   },
 
+  updateLikeCount(count) {
+    let current_actions_summary = this.get("actions_summary");
+
+    if (!this.actions_summary.find((entry) => entry.id === 2)) {
```

Comment:
What is ID 2 here? Can we have this in a named const or something so it's not a magic number?